configure.ac

```diff
@@ -578,6 +578,29 @@ if test "$WITH_ECONF" = "check"; then
 elif test "$WITH_ECONF" = "yes"; then
   PKG_CHECK_MODULES([ECONF], [libeconf >= 0.5.0], [ECONF_CFLAGS="-DUSE_ECONF=1 $ECONF_CFLAGS"], [])
 fi
+
+AC_MSG_CHECKING(for econf_readConfig)
```

Comment:
Please move this new econf-based check to `"$WITH_ECONF" = "yes"` branch.

Reply:
I have done it